examples/chainermn_simple.py

```diff
@@ -46,6 +47,10 @@ def create_model(trial):
 # FYI: Objective functions can take additional arguments
 # (https://optuna.readthedocs.io/en/stable/faq.html#objective-func-additional-args).
 def objective(trial, comm):
+    # The following line is required for CircleCI
```

Comment:
How about `The following line mitigates the memory problem in CircleCI`?
I don't think it's "required".

Reply:
You're right. I will use "The following line mitigates the memory problem in CircleCI".